homeassistant/components/ll_notify/README.md

```diff
@@ -0,0 +1,116 @@
+# Lovelace Notify / ll_notify
+
+ll_notify is a Home Assistant component that allows you to easily add notifications and alerts to a Lovelace dashboard.
+
+ll_notify is basically thin wrapper on top of the Javascript framework [AlertifyJS](https://alertifyjs.com/). It has a backend (python) component, auto-loads to the front-end, and has a JS wrapper in the Lovlace user interface. You can trigger a notification by calling a backend service. (eg: `call-service: ll_notify/success`)
+
+![screen recording](screenshot.gif)
+
+## Status: Alpha
+
+This is still in development mode. I'm trying to decide if I should do a PR to HACore (to become a builtin component) or if it should be a custom component.
```

Comment:
This would be perfect as a custom component

Reply:
@bramkragten 
Where are the toast messages and alerts (other than browser_mod?)

Im afraid I built this for nothing. Thats what I was looking for!  Thanks for a pointer.

Reply:
There is not a service call to show them, but the elements themselves are in the frontend repo, check `ha-toast` and `dialog-box`.

Reply:
@bramkragten Thanks for the info.

I'll publish this as a custom component, and then move on. I've spent too much time on this.

But two cents for you and the HA development team: you should really have good, configurable, attractive alerts in default Home Assistant. Right now if `ha-toast` isn't available as a service, or with documentation, it doesn't really exist. It's useless for most users. And the toasts I tried in `browser-mod` was a toy - one toast, no configuration, ugly. Not very useful. And `persistent_notification` is a very different use case. It does not give in your face, dashboard-ready information. 

Finally, doing this right is trivially easy, as my work here has shown. 95% of my effort was learning how to integrate with HA. 

2 cents for you guys. It's an easy win.